rxandroidble/src/main/java/com/polidea/rxandroidble2/RxBleDevice.java

```diff
@@ -109,6 +109,49 @@
      */
     Observable<RxBleConnection> establishConnection(boolean autoConnect, @NonNull Timeout operationTimeout);
 
+    /**
```

Comment:
Could you also extend the `OperationConnectTest` so it tests configurable timeouts? Currently it only has the `should emit BluetoothGattCallbackTimeoutException with a valid mac address on CallbackTimeout`

Reply:
@uKL the configurable connecting timeout is set at the `RxBleDevice` level. I am not sure how I can extend `OperationConnectTest`.

Reply:
It is set on `RxBleDevice` level but ultimately it ends in `ConnectOperation` and should be tested properly in `def "should emit BluetoothGattCallbackTimeoutException with a valid mac address on CallbackTimeout"` test in `OperationConnectTest:144` where there is a hardcoded value `35`. It should be verified that the timeout will happen exactly as specified in `OperationTimeoutConfiguration`